connector/pom.xml

```diff
@@ -160,17 +160,22 @@
 			<plugin>
 				<artifactId>maven-jar-plugin</artifactId>
 				<configuration>
+					<finalName>application</finalName>
```

Comment:
Can we rename this one to be dsp-true-connector or something that will be closer to the end product name?

Reply:
No problemo